shared/chat/conversation/input-area/normal/platform-input.desktop.tsx

```diff
@@ -363,7 +363,7 @@ const styles = Styles.styleSheetCreate(() => ({
     common: {
       ...Styles.globalStyles.flexBoxColumn,
       alignSelf: 'stretch',
-      backgroundColor: Styles.globalColors.black,
+      backgroundColor: Styles.isDarkMode ? Styles.globalColors.white : Styles.globalColors.black,
```

Comment:
This is the kind of thing we want to avoid

Reply:
I'm not sure how to deal with these special cases. Maybe create a new color in `globalColors` instead?